dags/enriched_surveillancegermes.py

```diff
@@ -140,4 +143,15 @@ def published_arguments(destination: str) -> List[str]:
             dag=dag,
         )
 
+        DATA_INTERVAL_END ='{{ data_interval_end | ds }}'
+        FILEDATE = DATA_INTERVAL_END.replace("-","_")
+        # pylint: disable=C0103
```

Comment:
Peut être mis tout en haut du fichier (ligne 4) avec les autres.

Reply:
done

Reply:
Tu peux faire tout en une ligne
`# pylint: disable=missing-function-docstring, duplicate-code, expression-not-assigned, C0103`
Mais en fait ça serait mieux de mettre le nom au lieu du code donc : 
`# pylint: disable=missing-function-docstring, duplicate-code, expression-not-assigned, invalid-name`